migrations/20200629154732-copy-entities-to-elasticsearch.js

```diff
@@ -118,6 +118,41 @@ module.exports = {
 
         entityObj["telemetry_entities"] = telemetryEntities;
 
+        let userData = await db.collection('userExtension').aggregate([
```

Comment:
@deepakr123 - We should rename this file also so as its executed automatically as a new migration.

Reply:
resolved